docs/differences_with_venv.rst

```diff
@@ -0,0 +1,17 @@
+Differences with venv
+=====================
+
+- ``venv`` is bundled with Python itself (starting from 3.3+), ``virtualenv``
+  is a third party package. This allows for ``virtualenv`` to have greater
+  flexibility when dealing with several Python versions.
+- ``virtualenv`` allows further :doc:`extendibility <extend>` to override default
+  functionality through a plugin system.
+- ``virtualenv`` can create virtual environments through its interpreter discovery
+  technique from arbitrary Python locations, they do not be installed or be on the
+  ``PATH``.
+- ``venv`` can only create virtual environments for the Python version it is
+  installed with, ``virtualenv`` supports both Python 2 and 3 and also other flavours
```

Comment:
You mention other flavours, but then you point out two we support. The other implies you already mentioned one and you're including new ones.

Reply:
I will rephrase this.